provider/win_driver_utils.py

```diff
@@ -27,7 +27,7 @@
                     "viofs": {"hwid": '"PCI\\VEN_1AF4&DEV_105A"', "device_name": "VirtIO FS Device"},
                     "viostor": {"hwid": '"PCI\\VEN_1AF4&DEV_1001" "PCI\\VEN_1AF4&DEV_1042"', "device_name": "Red Hat VirtIO SCSI controller"},
                     "vioscsi": {"hwid": '"PCI\\VEN_1AF4&DEV_1004" "PCI\\VEN_1AF4&DEV_1048"', "device_name": "Red Hat VirtIO SCSI pass-through controller"},
-                    "fwcfg": {"hwid": '"ACPI\\VEN_QEMU&DEV_0002"', "device_name": "QEMU FWCfg Device"}
+                    "fwcfg": {"hwid": '"ACPI\\QEMU0002"', "device_name": "QEMU FWCfg Device"}
```

Comment:
Is this correct also for 64bit drivers?

Reply:
Rerun the case with latest code.

(1/1) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.win_virtio_driver_update_test.with_fwcfg.before_bg_test.uninstall_install.q35: STARTED
 (1/1) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.win_virtio_driver_update_test.with_fwcfg.before_bg_test.uninstall_install.q35: PASS (716.32 s)